specs/capella/beacon-chain.md

```diff
@@ -317,6 +339,31 @@ def process_full_withdrawals(state: BeaconState) -> None:
             validator.fully_withdrawn_epoch = current_epoch
 ```
 
+#### Partial withdrawals
+
+*Note*: The function `process_partial_withdrawals` is new.
+
+```python
+def process_partial_withdrawals(state: BeaconState) -> None:
+    partial_withdrawals_count = 0
+    # Begin where we left off last time
+    validator_index = state.next_partial_withdrawal_index
+    for _ in range(len(state.validators)):
```

Comment:
sanity check:

- `MAX_PARTIAL_WITHDRAWALS_PER_EPOCH` = 256
- 225 epochs per day
- To process the current `state.validators`, it *at most* takes ~5.6 days to run a cycle right, right?

Reply:
yep, right about that